<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationToolbar</class>
 <widget class="QWidget" name="NavigationToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Orientation:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="positionComboBox">
     <item>
      <property name="text">
       <string>Selected Objects</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Parent Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Root</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Particular</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="particularButton">
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="orientationComboBox">
     <item>
      <property name="text">
       <string>Selected Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Parent Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Root</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="modeComboBox">
     <item>
      <property name="text">
       <string>Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Axes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
